dtc/PKGBUILD

```diff
@@ -8,6 +8,7 @@ arch=('i686' 'x86_64')
 url="https://git.kernel.org/pub/scm/utils/dtc/dtc/git"
 license=('GPL2')
 groups=('base')
+depends=('libyaml')
 makedepends=('python' 'swig' 'libcrypt-devel')
```

Comment:
needs libyaml-devel in makedepends

Reply:
Actually without 'libyaml-devel' in makedepends, the dtc still can be built successfully, but the weird thing is that the executable requires msys-yaml-0-2.dll.